redaxo/src/addons/be_style/plugins/customizer/assets/js/main.js

```diff
@@ -67,8 +67,13 @@ Customizer.init = function (container) {
         var mode = "application/x-httpd-php";
         var theme = rex.customizer_codemirror_defaulttheme;
 
+        if (document.body.classList.contains('rex-theme-dark')) {
```

Comment:
@christophboecker hier müsste wohl die Abfrage für den automatischen Darkmode angepasst werden.
Ich kann das unter Windows leider nicht testen. Automatisch wird bei mir gar nicht berücksichtigt.

Reply:
Ich bin kein Darkmode-Experte und lehne ihn sowieso grundsätzlich als unergonomischen Quatsch aus der IT-Steinzeit ab. Just my humble opinion, jeder wie er mag. Die Klasse auf dem Body sagt Dir nur, was Redaxo als Einstellung gerne hätte. Vorgabe Light/Dark/egal. Wenn egal greift die Systemeinstellung (im JS gesondert abfragen). Die von mir mal eingebrachte Lösung berücksichtigt nur diese Dark/Light/Egal-Logik mit genau den entsprechenden Theme-Namen. Wenn nun weitere Varianten, also andere Thema-Namen ins Spiel kommen, könnte ich nur vorschlagen, mal die [Lösung aus dem PR](https://github.com/redaxo/redaxo/pull/5025/files) von mir zu checken und entweder zu adaptieren oder zu erweitern. In den Files findest Du auf jeden Fall, wie die aktuelle Systemeinstellung angefragt wird und auch wie man Änderungen identifiziert und umschaltet